package.json

```diff
@@ -79,8 +81,31 @@
         "prune": "cd lib && ts-prune | grep -vE 'used in module| - default$| - package_version$|getSuperTokensRoutesForReactRouterDom$' && exit 1 || exit 0",
         "pretty-check": "npx pretty-quick --check .",
         "set-up-hooks": "cp hooks/pre-commit.sh .git/hooks/pre-commit && chmod +x .git/hooks/pre-commit",
-        "build-docs": "rm -rf ./docs && npx typedoc --out ./docs --tsconfig ./lib/tsconfig.json ./lib/ts/index.ts ./lib/ts/**/index.ts ./lib/ts/**/*/index.ts"
+        "build-docs": "rm -rf ./docs && npx typedoc --out ./docs --tsconfig ./lib/tsconfig.json ./lib/ts/index.ts ./lib/ts/**/index.ts ./lib/ts/**/*/index.ts",
+        "size": "size-limit"
     },
+    "size-limit": [
+        {
+            "path": "lib/build/index.js",
+            "limit": "5kb"
+        },
+        {
+            "path": "recipe/session/index.js",
+            "limit": "15kb"
+        },
+        {
+            "path": "recipe/thirdpartyemailpassword/index.js",
+            "limit": "110kb"
+        },
+        {
+            "path": "recipe/thirdparty/index.js",
+            "limit": "110kb"
+        },
+        {
+            "path": "recipe/emailpassword/index.js",
+            "limit": "110kb"
```

Comment:
Shouldn't we have lower limits on these compared to that of thirdpartyemailpassword?

Reply:
Also, will we have to add to this everytime we add a new recipe? If yes, you should edit the PR template's checklist for this repo to say to modify this (and any other script) if we add a new recipe.